docs/concepts/deployments.md

```diff
@@ -16,621 +16,182 @@ search:
-You can manage your deployments with a `prefect.yaml` file that describes how to prepare one or more [flow deployments](/concepts/deployments/). At a high level, you simply add the following file to your working directory:
-
-- [`prefect.yaml`](#the-prefect-yaml-file): a YAML file describing base settings for your deployments, procedural steps for preparing deployments, as well as instructions for preparing the execution environment for a deployment run
-
-You can initialize your deployment configuration, which creates the `prefect.yaml` file, by running the CLI command `prefect init` in any directory or repository that stores your flow code.
-
-!!! tip "Deployment configuration recipes"
-    Prefect ships with many off-the-shelf "recipes" that allow you to get started with more structure within your `prefect.yaml` file; run `prefect init` to be prompted with available recipes in your installation. You can provide a recipe name in your initialization command with the `--recipe` flag, otherwise Prefect will attempt to guess an appropriate recipe based on the structure of your working directory (for example if you initialize within a `git` repository, Prefect will use the `git` recipe).
-
-The `prefect.yaml` file contains deployment configuration for deployments created from this file, default instructions for how to build and push any necessary code artifacts (such as Docker images), and default instructions for pulling a deployment in remote execution environments (e.g., cloning a GitHub repository).
-
-Any deployment configuration can be overridden via options available on the `prefect deploy` CLI command when creating a deployment. 
-
-The base structure for `prefect.yaml` is as follows:
-
-```yaml
-# generic metadata
-prefect-version: null
-name: null
-
-# preparation steps
-build: null
-push: null
-
-# runtime steps
-pull: null
-
-# deployment configurations
-deployments:
-  - # base metadata
-    name: null
-    version: null
-    tags: []
-    description: null
-    schedule: null
-
-    # flow-specific fields
-    entrypoint: null
-    parameters: {}
-
-    # infra-specific fields
-    work_pool:
-      name: null
-      work_queue_name: null
-      job_variables: {}
-```
-
-The metadata fields are always pre-populated for you and are currently for bookkeeping purposes only.  The other sections are pre-populated based on recipe; if no recipe is provided, Prefect will attempt to guess an appropriate one based on local configuration.
-
-You can create deployments via the CLI command `prefect deploy` without ever needing to alter the `deployments` section of your `prefect.yaml` file — the `prefect deploy` command will help in deployment creation via interactive prompts. However, it is useful for version-controlling your deployments and managing multiple deployments.
+Deployments are server-side representations of flows. They store the crucial metadata needed for remote orchestration including _when_, _where_ and _how_ a workflow should run.
```

Comment:
```suggestion
Deployments are server-side representations of flows. They store the crucial metadata needed for remote orchestration including _when_, _where_, and _how_ a workflow should run.
```

Reply:
I believe we use the oxford comma everywhere else and i n style guide, so adding here.